lib/assertions.ts

```diff
@@ -730,14 +730,14 @@ export default class DOMAssertions {
    */
   hasPseudoElementStyle(
     selector: string | null,
-    expected: Dictionary<string>,
+    expected: Record<string, string>,
     message?: string
   ): DOMAssertions {
     let element = this.findTargetElement();
     if (!element) return this;
 
     let computedStyle = window.getComputedStyle(element, selector);
-    let expectedProperties = Object.keys(expected) as [keyof CSSStyleDeclaration];
```

Comment:
`[keyof CSSStyleDeclaration]` indicates an array with a single item of `keyof CSSStyleDeclaration`

Is that deliberate?